include/dev/umass.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Based on definitions contained in FreeBSD `sys/dev/usb/storage/umass.c`.
+ */
+#ifndef _DEV_UMASS_H_
+#define _DEV_UMASS_H_
+
+/* Bulk-Only features */
+
+#define UR_BBB_RESET 0xff       /* Bulk-Only reset */
+#define UR_BBB_GET_MAX_LUN 0xfe /* Get maximum lun */
+
+/* Command Block Wrapper */
+#define CBWCDBLENGTH 16
+
+typedef struct {
+  uint32_t dCBWSignature;
+  uint32_t dCBWTag;
+  uint32_t dCBWDataTransferLength;
+  uint8_t bCBWFlags;
+  uint8_t bCBWLUN;
+  uint8_t bCDBLength;
+  uint8_t CBWCDB[CBWCDBLENGTH];
```

Comment:
We prefer lowercase with underscore.

Reply:
It was consulted with @cahirwpz, and we've decided to conform to their convention throughout the presented headers.